examples/editor/components/Editor.jsx

```diff
@@ -1,6 +1,12 @@
 var React = require("react");
-var WebmakerLoginUX = require("./webmaker-login-ux/WebmakerLoginUX.jsx");
-var MakeEditor = require("./editor/MakeEditor.jsx");
+
+var WebmakerLoginUX;
+
+/* jshint ignore:start */
+WebmakerLoginUX = require("./../../../shared/components/webmaker-login-ux/WebmakerLoginUX.jsx");
+/* jshint ignore:end */
```

Comment:
this is an interesting quirk when it comes to browserify - this line is longer than 80 characters, but by refactoring the requirement out as a variable, browserify won't see it as a `require()` call that it can do anything with because the thing being required isn't a string. As such, I turn off linting to suppress the line length error for this single line (here, and in Gallery.jsx later on, too)